fix: Update README.md (#29)
anduril/lattice-sdk-cpp: README.md

## Requirements

> > [!WARNING]  
> > It's very important that the versions of libprotobuf match the version that it was compiled with as C++ requires very specific [guarantees](https://protobuf.dev/support/cross-version-runtime-guarantee/#cpp). The current requirements are:
> > * `gRPC == 1.68.0`
> > * `Protobuf == 29.0.0`
> > * `CMake >= 3.16`

Reviewer: Typically developers don't read anything written in callouts, so taking out this critical info out of the warning.
```suggestion
Install the required versions of the following:

* `gRPC == 1.68.0`
* `Protobuf == 29.0.0`
* `CMake >= 3.16`

C++ requires an exact version between `libprotobuf` and the runtime version at all times. For more information, see  [C++ specific guarantees](https://protobuf.dev/support/cross-version-runtime-guarantee/#cpp).
```

> > [!TIP]
> > As an alternative, we also provide the underlying Protobuf files in the `protos` directory if you wish to compile these yourselves with specific versions of Protobuf/gRPC.

Reviewer: Can we add a link to the protos directory?
```suggestion
Alternatively, use specific versions of Protobuf or gRPC by compiling the Protobuf files in the [`protos` directory](link).
```

Reviewer 2: +1, as permalink
